src/plugins/data_table/filter_view.js

```diff
@@ -56,7 +56,15 @@ var DataTableFilter = (function(options) {
 
   var StringFilterMenu = DataTableFilterMenu.extend({
     menuTemplate: _.template('\
-        <input class="filter-string" id ="value" type="text" placeholder="Search <%= title %>" />\
+      <div class="filterMenu filterMenu-string">\
+        <div class="filter-text">Show:</div>\
+        <div class="icon-addon addon-sm">\
+          <input class="filter-string form-control input-sm" id="value" type="text" name="filter-string" />\
+          <label for="filter-string" class="glyphicon glyphicon-search"></label>\
+        </div>\
+        <button class="btn btn-sm btn-filter" name="button" type="submit" title="">Apply</button>\
+        <button class="btn btn-primary btn-sm btn-clear pull-right" name="button" type="submit" title="">Clear</button>\
```

Comment:
having these 2 buttons repeated for each filter is not DRY, can you make this part of the base class (or at worst case, a shared child component that is rendered in place for each filter type)